database/artix7/settings.sh

```diff
@@ -1,26 +1 @@
-export XRAY_DATABASE="artix7"
```

Comment:
Why are we keeping this file? Are you afraid there will be a compatibility issue?

Reply:
Until I push a new rebuild of the database without `settings.sh` there will confusingly be a `database/xxx/settings.sh` and `settings/xxx.sh`. Having these files still exist means that your addition to the `download-database.sh` script will clobber them with the redirect.

Plus I thought it might confuse some people....

Reply:
Okay, we can leave for now, but lets trash this once we start pushing new DBs